autoscale/src/main/java/com/sequenceiq/periscope/controller/AutoScaleClusterCommonService.java

```diff
@@ -259,4 +274,11 @@ protected void createAutoscalingConfigChangedHistoryAndNotify(Cluster cluster) {
         notificationSender.sendHistoryUpdateNotification(
                 historyService.createEntry(scalingStatus, statusMessage, cluster), cluster);
     }
+
+    private boolean newOrPreExistingTimeAlerts(Cluster cluster, DistroXAutoscaleClusterRequest distroXAutoscaleClusterRequest) {
```

Comment:
This code is copied over from the validator class...

Reply:
Yes, it is the same as here also the same checks had to be done to used the same code.

Reply:
Removed